metanode/partition.go

```diff
@@ -501,6 +504,18 @@ type metaPartition struct {
 	versionLock            sync.Mutex
 }
 
+func (mp *metaPartition) IsForbidden() bool {
```

Comment:
why not use bool var directly.

Reply:
We need to use atomic variable.

Reply:
here, no need to user atomic var, because no concurrent change, and no effect in fact.

Reply:
> here, no need to user atomic var, because no concurrent change, and no effect in fact.

It seem that there are some code will write the variable when other threads are reading, are you sure atomic variable is unnecessary?